src/components/header/Header.jsx

```diff
@@ -0,0 +1,37 @@
+import './Header.scss';
+import React from 'react';
+
+export const Header = () => {
+  return (
+    <header className="header">
+      <div className="header__wrapper p-sm-vr">
+        <div className="header__title p-sm-hr">
```

Comment:
You use "__" and "--" in the class name, I suggest that be consistent with one of them so code readability will be better.

Reply:
used BEM naming convention 
--Modifier
__element